Hymns/Display/SplashScreenView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+import Lottie
+
+//Implemented using https://www.raywenderlich.com/4503153-how-to-create-a-splash-screen-with-swiftui
+//And the help from https://www.youtube.com/watch?v=fVehE3Jf7K0
+struct SplashScreenView: UIViewRepresentable {
+    let animationView = AnimationView()
```

Comment:
Just curious, why is this an instance var instead of just a local variable? Does makeUIView get called multiple times and we need the same instance of animation view?

Reply:
Changed to local var. Good point.